articles/azure/azs-how-add-data-managed-portal.md

```diff
@@ -0,0 +1,113 @@
+---
+title: How to add a managed disk to a virtual machine using UKCloud Azure Stack portal | UKCloud Ltd
+description: Provides help adding managed disks to a virtual machine on UKCloud for Microsoft Azure Stack
+services: azure-stack
+author: Shaion O'Niel
+
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2: Add a managed disk to a virtual machine
+toc_sub3:
+toc_sub4:
+toc_title: Add a managed disk to a virtual machine - portal
+toc_fullpath: Users/How To/
+toc_mdlink: azs-how-add-managed-disks-portal.md
+---
+
+# How to add a managed disk to a virtual machine using UKCloud Azure Stack portal
+
+## Overview
+
+UKCloud for Azure Stack supports the use of managed disks on virtual machines. Managed disks can be used as both OS disks and a data disk.
+For more information, see [Introduction to Azure managed disks](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/managed-disks-overview).
+
+The following process shows you how to add a managed disk to a virtual machine using the UKCloud for Azure Stack Portal.
+
+## Intended audience
+
+To complete the steps in this guide, you must have appropriate access to a subscription in the Azure Stack portal.
+
+## Adding a disks to an existing virtual machine in azure stack
```

Comment:
Change to:
Adding a disk to an existing virtual machine using the Azure Stack portal